src/main/java/software/amazon/cloudwatchlogs/emf/environment/EnvironmentProvider.java

```diff
@@ -35,7 +35,7 @@
     // Ordering of this array matters
     private final Environment[] environments =
             new Environment[] {
-                lambdaEnvironment, ec2Environment, ecsEnvironment, defaultEnvironment
+                lambdaEnvironment, ecsEnvironment, ec2Environment, defaultEnvironment
```

Comment:
Does ECS respond to the metadata endpoint request when running non-Fargate tasks?